curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e63bb1e32d23b6adbe44.md

```diff
@@ -271,8 +271,7 @@ function calculateCalories(e) {
 
   const consumedCalories = breakfastCalories + lunchCalories + dinnerCalories + snacksCalories;
   const remainingCalories = budgetCalories - consumedCalories + exerciseCalories;
-  const surplusOrDeficit = remainingCalories >= 0 ? 'Surplus' : 'Deficit';
-  output.innerHTML = ``;
+	const surplusOrDeficit = remainingCalories < 0 ? 'Surplus' : 'Deficit';  output.innerHTML = ``;
```

Comment:
not sure if this is an indentation issue caused by me. however if its an issue i can go back fix these

Reply:
The indentation should be resolved.

But also output.innerHTML=``  should not be right next it to it either. It should be below it